django_mongodb/query.py

```diff
@@ -51,28 +50,11 @@ def __init__(self, compiler):
         self.lookup_pipeline = None
         self.project_fields = None
         self.aggregation_pipeline = compiler.aggregation_pipeline
+        self.extra_fields = None
```

Comment:
I think now we can pass the compiler alone and the query will know how to fill in all the fields from the compiler.

Reply:
This is an idea for a later cleanup, right?

Reply:
yes, to avoid doing 
```python
query.something = value
query.anotherthing = other_value
...
```